Buildings/Controls/OBC/CDL/Logical/Validation/ConstantDutyCycle.mo

```diff
@@ -0,0 +1,92 @@
+within Buildings.Controls.OBC.CDL.Logical.Validation;
+model ConstantDutyCycle
```

Comment:
It would be more user friendly to call these `DutyCycleVariable` and `DutyCycleConstant` as then, they will be next to each other in a menu that allows selecting blocks.